src/librustdoc/clean/mod.rs

```diff
@@ -2272,13 +2287,30 @@ impl Clean<Vec<Item>> for doctree::Import<'_> {
             }
             if !denied {
                 let mut visited = FxHashSet::default();
-                if let Some(items) =
+                if let Some(mut items) =
                     inline::try_inline(cx, path.res, name, Some(self.attrs), &mut visited)
                 {
+                    // In case this is a macro, we don't want to show the reexport, only the macro
+                    // itself.
```

Comment:
Can you give an example of the code this is describing? Is this like an import expanded from a macro?

Reply:
When you reexport a macro, we don't show the reexport, just the macro. As simple as that. :)

Reply:
Oh oh I see what you mean - the macro is being inlined, so we don't need to show both the `use` and the macro.